### ISSUE TITLE
(GSSOC21) Traveling Salesman Problem in Python using Backtracking

### ISSUE THREAD

**Issue by batul02** [Algo-Phantoms/Algo-Tree#1197]
**Is your feature request related to a problem? Please describe.**
This is a Travelling Salesman Problem which I am willing to do using Backtracking algorithm.

**Describe the solution you'd like**
This Problem returns the shortest path for the travelling salesman who will visit every nodes he has to and returns to the node where he started from.

**Do you want to work on it**
I am a GSSOC'21 participant. I would like to work on this issue. I am comfortable with python and have a decent knowledge of this algorithm.

**Comment by subodhsiddharth** [comment]
I am a GsSOCs'21 participants can I do same in CPP or C.

**Comment by batul02** [comment]
Hi @subodhsiddharth, the mentors are not responding to the issue. I don't think we can make a PR without them assigning the task to us. Or should I create PR? Because I have been waiting for them to assign this issue to me.

**Comment by subodhsiddharth** [comment]
Yes @batul02 . Now we can do it and see our progress update on https://gssoc.girlscript.tech/profile.html .
